src/lib.rs

```diff
@@ -621,6 +701,12 @@ impl ModemDeactivation {
                 // Turn off the UICC
                 at::send_at::<0>("AT+CFUN=40").await?;
```

Comment:
This needs to be looked at. When we shut off the LTE, but have a uicc on, we don't want to do this.
This can maybe be solved most elegantly by having the LteLink contain a UiccLink. Then each can manager their own

Reply:
Well yeah, but the links are ZST's, so in terms of RAM usage it doesn't really matter. There's only gonna be some extra function calls and one extra AT command to turn the Uicc on before the LTE, which is probably fine

Reply:
Even then, we could have a private constructor for the UiccLink only used by LTE link that skips that AT command

Reply:
I did it differently now, just doing two deactivations as needed based on the counts. What do you think?